plugin/storage/grpc/options.go

```diff
@@ -28,21 +28,21 @@ func tlsFlagsConfig() tlscfg.ClientFlagsConfig {
 }
 
 // AddFlags adds flags for Options
-func v1AddFlags(flagSet *flag.FlagSet) {
+func AddFlags(flagSet *flag.FlagSet) {
```

Comment:
does it need to be public?

Reply:
they do not - made this one and the one below private! thank you